pool/src/pool.rs

```diff
@@ -268,7 +269,12 @@ impl Pool {
 		header: &BlockHeader,
 	) -> Result<BlockSums, PoolError> {
 		let overage = tx.overage();
-		let offset = (header.total_kernel_offset() + tx.offset.clone())?;
+
+		let offset = {
+			let secp = static_secp_instance();
```

Comment:
So it is my understanding (also related to previous comment), that you want to be able to choose the secp instance ?

Reply:
We want to pass an secp instance in, just to keep the api consistent with the rest of the `BlindingFactor` fns. We don't _need_ to do this but I think it makes it cleaner.

Reply:
This impl detail was hidden behind the `Add` trait via the `A + B` syntax and I think this hid too much (in terms of needing to obtain an secp instance).

Reply:
I see thanks for the clarification. 👍